nat-lab/docker-compose.yml

```diff
@@ -508,6 +509,24 @@ services:
         ipv4_address: 10.0.80.83
         ipv6_address: 2001:db8:85a4::adda:edde:8
 
+  # A host which drop all packets with size larger than 1300
```

Comment:
a tiny tiny improvement: could we pass 1300 via entrypoint?

Reply:
Um, probably. The packets are dropped with the help of iptables configuration. Making it not hardocded would increase the complexity without benefit IMHO

Reply:
Wouldn't it be just passing `$1` instead of `1300` in the entrypoint script?

Reply:
Yeah, but what's the point? Presumably, no one will ever change this